models/core/eligibility.sql

```diff
@@ -0,0 +1,92 @@
+-----------------------------------------------------------------------------------------------------------------
+-- Author       Thu Xuan Vu
+-- Created      August 2022
+-- Purpose      Create eligibility file at the month grain.
+-----------------------------------------------------------------------------------------------------------------
+-- Modification History
+--
+-----------------------------------------------------------------------------------------------------------------
+
+with dual_status as(
+  select 
+    desy_sort_key 
+    ,right(month,2) as month
+    ,REFERENCE_YEAR as year

```

Comment:
Minor suggestion to use the same case formatting throughout this model's code.

```suggestion
    ,reference_year as year

```